x-pack/examples/reporting_example/public/application.tsx

```diff
@@ -9,7 +9,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Router, Route, Switch } from 'react-router-dom';
 import { AppMountParameters, CoreStart } from '../../../../src/core/public';
-import { KibanaThemeProvider } from '../../../../../kibana/src/plugins/kibana_react/public';
```

Comment:
The old import wasn't working for me and I think it was an error anyway because it was stepping out of the repo root, then back in (i'm using git worktree, so my kibana root is _not_ the directory `kibana`)

Reply:
Nice spot